bamboo/set-bamboo-env-variables.sh

```diff
@@ -51,6 +51,11 @@ done
 ## Get the current git SHA
 export GIT_SHA=$(git rev-parse HEAD)
 
+## Always set GIT_PR true if master branch
+if [[ $BRANCH == master ]]; then
```

Comment:
This may not work because the `if [[ -z $BRANCH ]]; then` conditional is at L70 and there `$BRANCH` may not be defined yet.